Marketplace/Cardz/Card.js

```diff
@@ -78,28 +86,8 @@
         var props = Entities.getEntityProperties(_this.entityID);

```

Comment:
Only get the properties that you need from the entity in all cases of Entities.getEntityProperties